File: app/src/main/java/com/yhjoo/dochef/adapter/ReviewListAdapter.java
package com.yhjoo.dochef.adapter;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.BaseViewHolder;
import com.yhjoo.dochef.R;
import com.yhjoo.dochef.model.Review;
import com.yhjoo.dochef.utils.Utils;

public class ReviewListAdapter extends BaseQuickAdapter<Review, BaseViewHolder> {
    public ReviewListAdapter() {
        super(R.layout.li_review);
    }

    @Override
    protected void convert(BaseViewHolder helper, Review item) {
        helper.setText(R.id.review_nickname, item.getNickname());
        helper.setText(R.id.review_contents, item.getContents());
        helper.setText(R.id.review_datetext, Utils.convertMillisToText(item.getDateTime()));
        helper.setRating(R.id.review_rating, item.getRating());
        helper.addOnClickListener(R.id.review_user_wrapper);
    }
}

File: app/src/main/java/com/yhjoo/dochef/ui/adapter/ReviewListAdapter.kt
package com.yhjoo.dochef.ui.adapter

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.BaseViewHolder
import com.yhjoo.dochef.R
import com.yhjoo.dochef.model.Review
import com.yhjoo.dochef.utilities.Utils

class ReviewListAdapter : BaseQuickAdapter<Review, BaseViewHolder>(R.layout.li_review) {
    override fun convert(helper: BaseViewHolder, item: Review) {
        helper.setText(R.id.review_nickname, item.nickname)
        helper.setText(R.id.review_contents, item.contents)
        helper.setText(R.id.review_datetext, Utils.convertMillisToText(item.dateTime))
        helper.setRating(R.id.review_rating, item.rating.toFloat())
        helper.addOnClickListener(R.id.review_user_wrapper)
    }
}